### ISSUE TITLE
Add ability to exclude all js errors by pattern

### ISSUE THREAD

**Issue by flaminestone** [onlyoffice-testing-robot/onlyoffice_documentserver_testing_framework#34]
Sometimes, I need to exclude some errors, like all errors about image missing in files.
Can you add environment variable with excluding, like this:

```bash
export ODTF_ERRORS_EXCLUDE=["error1", 'error2']
```

Is a template for variable for environment

**Comment by ShockwaveNN** [comment]
Array cannot be stored in enviroment variables, so I'm thinking something like this:

```
export ODTF_ERRORS_EXCLUDE="error1 error2"
```

But in that case space cannot be in ignored errors.
Is that all right?

**Comment by flaminestone** [comment]
@ShockwaveNN I think, it is not good idea, because most of errors is contains spaces, reformatting can be difficalt. May be, to keep json with errors?
```bash
export ODTF_ERRORS_EXCLUDE="{"error1_name":"error1","error2_name":"error2"}"
```


```

**Comment by ShockwaveNN** [comment]
Do you really need those names for errors? Like `error1_name`
 
Also ODTF_ERRORS_EXCLUDE is not a good name, it's not univerasl

I'm thinking something like this:

```
export ONLYOFFICE_WEBDRIVER_OPTIONS='{ "IgnoredJSErrors" : ["pptx_unpacked/ppt/drawings/NULL - Failed to load resource", "some error happend and this is not my cause"]}'
```

**Comment by flaminestone** [comment]
Yes, it wiil be good to make this feature

**Comment by ShockwaveNN** [comment]
Implemented in #35 
Example of usage in spec:
https://github.com/onlyoffice-testing-robot/onlyoffice_documentserver_testing_framework/blob/17285530e31803473b35e34cb65f124e3dda2f58/spec/test_instance_docs/env_options_ignored_errors_spec.rb
